[2024-08-21T20:43:40.225+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-21T20:43:40.245+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:43:37.777131+00:00 [queued]>
[2024-08-21T20:43:40.251+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:43:37.777131+00:00 [queued]>
[2024-08-21T20:43:40.251+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-21T20:43:40.266+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): init> on 2024-08-21 15:43:37.777131+00:00
[2024-08-21T20:43:40.278+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=302223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T20:43:40.279+0500] {standard_task_runner.py:72} INFO - Started process 302275 to run task
[2024-08-21T20:43:40.278+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag_home', 'init', 'manual__2024-08-21T15:43:37.777131+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/homework_stepik-checkpoint.py', '--cfg-path', '/tmp/tmph_qs3yli']
[2024-08-21T20:43:40.280+0500] {standard_task_runner.py:105} INFO - Job 169: Subtask init
[2024-08-21T20:43:40.338+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:43:37.777131+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-21T20:43:40.409+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag_home' AIRFLOW_CTX_TASK_ID='init' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T15:43:37.777131+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-21T15:43:37.777131+00:00'
[2024-08-21T20:43:40.411+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-21T20:43:40.645+0500] {migration.py:216} INFO - Context impl PostgresqlImpl.
[2024-08-21T20:43:40.646+0500] {migration.py:219} INFO - Will assume transactional DDL.
[2024-08-21T20:43:40.697+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240821 20:43'}
[2024-08-21T20:43:40.723+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-21T20:43:40.724+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag_home, task_id=init, run_id=manual__2024-08-21T15:43:37.777131+00:00, execution_date=20240821T154337, start_date=20240821T154340, end_date=20240821T154340
[2024-08-21T20:43:40.773+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-21T20:43:40.801+0500] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21T20:43:40.803+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
